.travis.yml

```diff
@@ -7,7 +7,17 @@ r_packages:
 
 after_success:
   - Rscript -e 'library(covr); codecov()'
-  
+  - Rscript -e 'pkgdown::build_site()'
+
+deploy:
```

Comment:
To test that the website was rebuilt correctly, let's change the vignette URL in the readme.  What do you think about changing the vignette URL from https://cran.r-project.org/web/packages/LPWC/vignettes/LPWC.html to https://gitter-lab.github.io/LPWC/articles/LPWC.html ?

Reply:
sure. That works for me.